api/python/tests/integration/test_packages.py

```diff
@@ -1410,3 +1424,68 @@ def test_resolve_hash(self):
 
         with pytest.raises(QuiltException, match='Found multiple matches'):
             Package.resolve_hash(pkg_name, LOCAL_REGISTRY, hash_prefix)
+
+
+# The following tests were moved out of the PackageTest class to enable parametrization.
+# see (https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses)
+@pytest.mark.parametrize(
+    'update_policy, expected_one_byte',
+    [
+        ('incoming', b'one'),
+        ('existing', b'1')
```

Comment:
In addition to the default argument, you can add an xfail test case, e.g., `"bad_policy"` [like this](https://github.com/quiltdata/quilt/blob/master/lambdas/thumbnail/tests/test_thumbnail.py#L57).